src/main/kotlin/no/nav/dagpenger/inntekt/subsumsjonbrukt/KafkaSubsumsjonBruktDataConsumer.kt

```diff
@@ -76,7 +76,11 @@ internal class KafkaSubsumsjonBruktDataConsumer(
                         val records = consumer.poll(Duration.ofMillis(100))
                         val ids = records.asSequence()
                             .map { record -> record.value() }
-                            .map { packet -> InntektId(packet.getMapValue("faktum")["inntektsId"] as String) }
+                            .filter { packet -> packet.hasField("faktum") }
```

Comment:
Må si APIet til rapids-and-rivers er mer sexy enn Streams ass.

Reply:
Enig, vurderte en liten stund å rapidify denne, men blææ